interop_binaries/Cargo.toml

```diff
@@ -51,6 +51,10 @@ tokio = { version = "1.26", features = ["full", "tracing"] }
 tracing = "0.1.37"
 tracing-log = "0.1.3"
 tracing-subscriber = { version = "0.3", features = ["std", "env-filter", "fmt"] }
+trillium = "0.2.8"
+trillium-api = { version = "0.2.0-rc.2", default-features = false }
+trillium-router = "0.3.5"
+trillium-tokio = "0.2.1"
```

Comment:
Sorta orthogonal to this change, but: let's put all the trillium deps in the workspace `Cargo.toml` and reference those from each crate.